modules/layout-renderer.twig

```diff
@@ -31,14 +31,10 @@
         <meta name="public-path" content="{{ theme.rootUri() }}"/>
         
         <link rel="shortcut icon" href="{{ theme.uri("img/favicon.ico") }}" />
-        <link rel="stylesheet" type="text/css" href="{{ theme.rootUri() }}dist/preview/html-preview.css" />
-        <link rel="stylesheet" type="text/css" href="{{ theme.uri("css/html-preview.css") }}" />
```

Comment:
This file is used during the white label process to replace the Xibo logo with the white label logo equivalent, which I guess is something we need to handle in XLR?

Reply:
Yes, I guess we can have this handled by XLR instead. I just need to know what data determines if it's white label or not.